### ISSUE TITLE
Show page number on Section heading?

### ISSUE THREAD

**Issue by CarlosNZ** [openmsupply/application-manager-web-app#740]
I noticed that in Drug registration template the page numbers are hard-coded into the template as TextInfo elements.

Not sure this is a good way to handle this (since page numbering could change, for one). If we feel the page numbers are important enough to display (other than in the Progress bar) we should probably just add them to the Section heading instead of expecting the template to present them. So it would say something like:
"SECTION 3 — PAGE 2" instead of where it currently just says "SECTION 3"

What do we think?

**Comment by andreievg** [comment]
I am not sure if it's important to show page number at all (unless it has some meaning, as per below)

I had a thought the other day, it goes back to Valerio wanting to 'group' sets of form elements, and him question the need for pages. Should we be naming pages ? This could be done on the page break element.  Could possibly be problematic if page name is too long (with the left side progress bar, progress indicators)

**Comment by CarlosNZ** [comment]
> I am not sure if it's important to show page number at all (unless it has some meaning, as per below)

I agree, just that if we ARE going to show it, it should be handled by the UI, not hard-coded into template.

> I had a thought the other day, it goes back to Valerio wanting to 'group' sets of form elements, and him question the need for pages. Should we be naming pages ? This could be done on the page break element. Could possibly be problematic if page name is too long (with the left side progress bar, progress indicators)

Yeah, that could be an optional field on PageBreak element. If present, show page name, if not, show nothing. Don't think length would be an issue -- wouldn't we just show it in the main form column under (or next to) the Section name?

**Comment by nmadruga** [comment]
> Yeah, that could be an optional field on PageBreak element. If present, show page name, if not, show nothing. Don't think length would be an issue -- wouldn't we just show it in the main form column under (or next to) the Section name?

I don't like the idea of adding this property to the PageBreak it self since this goes after all the element in a page and adding a title that would display on the top seems misleading... I used to set this "group" title on the `textInfo` that goes on the top of the page and though this was a nice way to have a title, but that doesn't show in the progress bar. Not sure it we should change that, anyway if we will be changing it would be better to go back to that idea of having pages in the db
